test/images/jessie-dnsutils/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=debian:jessie
```

Comment:
Know this was the case before, but wonder if `debian-base` would work here @ixdy ?